src/js/libs.js

```diff
@@ -1,7 +1,10 @@
 import Alpine from 'alpinejs';
 import { queryAPI } from './api';
+import submissionsTable from './page-table.js';
 
 window.Alpine = Alpine;
+Alpine.data('submissionsTable', submissionsTable);
```

Comment:
Did this not work with an `x-data` attribute in the markup on the table? Ideally, it would be good to keep this out of the script that gets loaded everywhere.

That said, I couldn't get the store to work that way (which is why _that_ is in the script that gets loaded everywhere) so if you need to leave it there, that's fine.

Reply:
I needed to add that in before the Alpine.start() call.